.github/workflows/move-to-next-iteration.yml

```diff
@@ -0,0 +1,22 @@
+on:
+  schedule:
+    # Runs every Wednesday at 2:00 AM UTC
+    # https://crontab.guru/#0_2_*_*_3
+    - cron: "0 2 * * 3"
+
+jobs:
+  move-to-next-iteration:
+    name: Transfer tasks from the previous iteration to the current iteration.
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Move issues to next iteration
+        uses: blombard/move-to-next-iteration@master
+        with:
+          owner: input-output-hk
+          number: 102
```

Comment:
Shouldn't we move for all boards, 146 (Discovery) and 148 (Planning) as well?

Reply:
@stevenj 
The purpose of this PR is to address the ongoing issue developers face moving WIP tasks to the current iteration for project 102. Does this problem exist for 146 and 148?